pom.xml

```diff
@@ -34,21 +34,39 @@
 
     <!-- runtime dependencies -->
     <folio-spring-base.version>7.2.2</folio-spring-base.version>
+    <folio-spring-system-user.version>7.2.2</folio-spring-system-user.version>
     <openapi-generator.version>6.2.1</openapi-generator.version>
     <mapstruct.version>1.5.3.Final</mapstruct.version>
 
     <!-- test dependencies -->
     <wiremock.version>3.2.0</wiremock.version>
-    <testcontainers.version>1.19.3</testcontainers.version>
+    <testcontainers.version>1.17.6</testcontainers.version>
     <awaitility.version>4.2.0</awaitility.version>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>testcontainers-bom</artifactId>
+        <version>${testcontainers.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>folio-spring-base</artifactId>
       <version>${folio-spring-base.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-spring-system-user</artifactId>
+      <version>${folio-spring-system-user.version}</version>
```

Comment:
folio-spring-base and folio-spring-system-user are modules of the same project: https://github.com/folio-org/folio-spring-support

Perhaps it makes sense to keep their versions aligned by using the same variable for the version?